app/javascript/react/screens/App/Overview/components/AggregateCards/ActiveTransformationPlans/ActiveTransformationPlans.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import {
+  Icon,
+  Card,
+  AggregateStatusCount,
+  AggregateStatusNotifications,
+  AggregateStatusNotification,
+  Spinner
+} from 'patternfly-react';
+
+const ActiveTransformationPlans = ({ activePlans, loading }) => {
+  const countDescription =
+    activePlans.length === 1
+      ? 'Migration Plan In Progress'
+      : 'Migration Plans In Progress';
+
+  const erroredPlans = activePlans.filter(plan => {
+    const [mostRecentRequest] = plan.miq_requests.slice(-1);
+    return mostRecentRequest.miq_request_tasks.some(
+      task =>
+        task.options.progress.current_description ===
```

Comment:
for now - and per Bill... the task is failed if it's task.options.progress.current_description !== ''Virtual machine migrated'. This was the last direction I was given on this.

Reply:
Ok, I guess we can leave it with a placeholder for now.  We need the exact string match, b/c otherwise this filter will match on all the task descriptions (depending on which `/state` its in) like [here](https://github.com/priley86/miq_v2v_ui_plugin/pull/196/files#diff-9b2adb88683df284c00f835084862fd7R80)

Reply:
ok - i agree. we will need to know the exact failure messages to look out for, otherwise it will be shown as error immediately. will ask @bzwei again...

Reply:
Would `task.status === 'error'` if it was not done successfully?  @bzwei

Reply:
ah, I guess we could check that `task.status !== 'Ok'` for failure